### ISSUE TITLE
Be more clear about where Filter-Rule or Filter-Id AVPs go.

### ISSUE THREAD

**Issue by dcdolson** [lbertz02/rfc4006bis#23]
Regarding this paragraph in section "Restrict Access Action":

```
   A QoS-Final-Unit-Indication AVP with the Final-Unit-Action
   RESTRICT_ACCESS indicates to the device supporting this action that,
   upon consumption of the final granted units, the actions specified in
   Filter-Rule AVP(s) MUST restrict the traffic according to the
   classifiers in the Filter-Rule AVP(s).  If Filter-Id AVP(s) are
   provided in the Credit-Control-Answer message, the credit control
   client MUST restrict the traffic according to the IP packet filters
   identified by the Filter-Id AVP(s).  The credit-control server SHOULD
   include either the Filter-Rule AVP or the Filter-Id AVP in the
   Credit-Control-Answer message.
```

I think the last sentence could be clearer vs. saying "in the CCA message."
Should it be: `The credit-control server SHOULD include either the Filter-Rule AVP or the Filter-Id AVP in the QoS-Final-Unit-Indication AVP of the Credit-Control-Answer message.`

**Comment by yuvalif** [comment]
then the same fix should be done for the previous paragraph:

>  A Final-Unit-Indication AVP with the Final-Unit-Action
   RESTRICT_ACCESS indicates to the device supporting this action that,
   upon consumption of the final granted units, the user's access MUST
   be restricted according to the IP packet filters given in the
   Restriction-Filter-Rule AVP(s) or according to the IP packet filters
   identified by the Filter-Id AVP(s).  The credit-control server SHOULD
   include either the Restriction-Filter-Rule AVP or the Filter-Id AVP
   in the Credit-Control-Answer message.

**Comment by dcdolson** [comment]
Fixed in commit 28b90f9ed9f77ee67c0f41ccf5fd8655baf80ebe
